components/clientComponents/forms/Review/Review.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+import { Button } from "@clientComponents/globals";
+import { useTranslation } from "@i18n/client";
+import { useFocusIt } from "@lib/hooks/useFocusIt";
+import { useGCFormsContext } from "@lib/hooks/useGCFormContext";
+import { getLocalizedProperty } from "@lib/utils";
+import { useRef } from "react";
+
+export const Review = (): React.ReactElement => {
+  const {
+    t,
+    i18n: { language: lang },
+  } = useTranslation("review");
+  const { groups, getValues, setGroup, formRecord } = useGCFormsContext();
+  const formValues = getValues();
+  const headingRef = useRef(null);
+  useFocusIt({ elRef: headingRef });
+
+  // Remove the review and end groups so it won't show as a group in the review page
+  const groupsTemp = { ...groups };
```

Comment:
Thinking either maybe on 29 we can filter these out vs create a new obj and deleteing

---

and or maybe rename the var to something like reviewGroups or displayGroups vs groupsTemp

Reply:
I went back and forth on this. I was thinking the delete syntax was more clear and obvious about what's going on. But on second thought, most of our code uses the functional programming way with filter() so that makes more sense. Will change